test/test_ops.py

```diff
@@ -1774,8 +1779,7 @@ def test_refs_are_in_decomp_table(self, op):
     "nn.functional.pixel_unshuffle",
 )
 
-fake_striding_skips = (
-    "diag_embed",
+fake_striding_skips = [
```

Comment:
Can this add comments for each of these objects describing what they do and how we should maintain them?